src/js/pages/attendee/NewsFeed.jsx

```diff
@@ -1,130 +1,210 @@
 import React from "react";
-
 import NewsFeedPost from "components/NewsFeedPost";
 import NewsFeedStore from "stores/NewsFeedStore";
-// import style from 'sass/newsfeed.scss';
+import Dropzone from 'react-dropzone';
+import MdHighlightRemove from 'react-icons/lib/md/highlight-remove';
+import MdPhotoCamera from 'react-icons/lib/md/photo-camera';
+import TiSocialFacebook from 'react-icons/lib/ti/social-facebook';
+import TiSocialTwitter from 'react-icons/lib/ti/social-twitter';
+// import style from '../../sass/newsfeed.scss';
 
+// TODO: LIMIT NUMBER OF NEWLINES?
 
 export default class NewsFeed extends React.Component {
 
-	constructor() {
+  constructor() {
     super();
     this.getNewsFeedPosts = this.getNewsFeedPosts.bind(this);
```

Comment:
Is it already that style in other places? I thought I was following convention. Otherwise we can do 1 app wide change for all instances of this